recipes/termcap/all/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+cmake_minimum_required(VERSION 2.8.11)
```

Comment:
Why do you provide a custom cmakelist for this library?

Reply:
To support Visual Studio

Reply:
Was this extracted from anywhere?

Reply:
good question, I made the original cmake file in Bincrafters: https://github.com/bincrafters/conan-termcap/blob/testing/1.3.1/CMakeLists.txt. However, this one I dunno where came from.

Reply:
it's based only based on bincrafters' recipe.